src/lib.rs

```diff
@@ -1,21 +1,21 @@
 #[cfg(target_os = "macos")]
-extern crate sapp_darwin as sapp;
+pub extern crate sapp_darwin as sapp;
 #[cfg(not(any(
     target_os = "linux",
     target_os = "macos",
     target_os = "android",
     target_arch = "wasm32",
     windows
 )))]
-extern crate sapp_dummy as sapp;
+pub extern crate sapp_dummy as sapp;
```

Comment:
I am seriously concerned about exposing internal gl functions to users. 

I understand that this may be helpful for platform-dependent extensions for miniquad, but there are a couple of problems:
- miniquad use a lot of internal caches and it is really easy to break gl state by raw gl calls
- sapp-* crates are not designed as a general purpose GL loaders 
- both problems will be even more serious with multiple graphics backends in the future

Reply:
hmm, that's a good point. Do you mean for example that Metal support will not work? Would the solution be to update Context to support everything nanovg uses in such case? I like having the GL calls, because I can iterate on my graphics library quickly, but I see the good option of being more platform agnostic.

Reply:
If it is possible - it is better to expose those functions from the context for sure. 
Is it really much of things missing? Maybe I can help with some of those things? 

I can give an example - ColorMask should belong to miniquad's Pipeline. 
And it even is in PipelineParams - as `color_write`. But the actual call to glColorMask was unimplemented.
I guess it is the same for other functions you had to expose from gl.

Reply:
WIP `color_mask` PR: https://github.com/not-fl3/miniquad/pull/49 :)

Reply:
Nice! I will look again at the source and make a list of things missing in Context, and see how long will it take to move them in Context.

Reply:
For this PR, I propose to remove the public crate sapp::* and still keep the other benefits. Will make an update to the PR and push today/tomorrow.